packages/dds/tree/src/shared-tree-core/sharedTreeCore.ts

```diff
@@ -69,13 +73,13 @@ export interface ClonableSchemaAndPolicy extends SchemaAndPolicy {
 export class SharedTreeCore<TEditor extends ChangeFamilyEditor, TChange> extends SharedObject {
 	private readonly editManager: EditManager<TEditor, TChange, ChangeFamily<TEditor, TChange>>;
 	private readonly summarizables: readonly Summarizable[];
-
 	/**
 	 * The sequence number that this instance is at.
 	 * This number is artificial in that it is made up by this instance as opposed to being provided by the runtime.
 	 * Is `undefined` after (and only after) this instance is attached.
 	 */
 	private detachedRevision: SeqNumber | undefined = minimumPossibleSequenceNumber;
+	rebaseLogger: ITelemetryLoggerExt;
```

Comment:
We don't need this

Reply:
I'm currently building my code with format and necessary changes. Just one second!